crates/graphql/src/schema/objects/wallet.rs

```diff
@@ -427,20 +430,20 @@ impl Wallet {
         &self,
         ctx: &AppContext,
         reward_center: PublicKey<RewardCenter>,
-    ) -> FieldResult<U64> {
+    ) -> FieldResult<I64> {
         let db_conn = ctx.shared.db.get()?;
 
-        let wallet_total_rewards = wallet_total_rewards::table
+        let wallet_total_reward = wallet_total_rewards::table
             .select(wallet_total_rewards::total_reward)
             .filter(
                 wallet_total_rewards::reward_center_address
                     .eq(&reward_center)
                     .and(wallet_total_rewards::wallet_address.eq(&self.address)),
             )
             .first::<BigDecimal>(&db_conn)
-            .context("Failed to load total rewards")?;
+            .unwrap_or_else(|_| BigDecimal::zero());
```

Comment:
```rust
.optional()
.first::<BigDecimal>(&db_conn)
.unwrap_or(BigDecimal::zero())
```

This should be U64 why the switch to I64?

Reply:
This was after the last review. I supposed that it is required to be I64 - https://github.com/holaplex/indexer/pull/863#discussion_r1010360723

Reply:
Does this work

```rust
.first::<BigDecimal>(&db_conn)
.optional()
.unwrap_or(BigDecimal::zero())
```

Reply:
This will throw error - `expected Option<BigDecimal>, found BigDecimal`

Reply:
This is what actually worked:
```
.optional()
.transpose()
.unwrap_or_else(|| Ok(BigDecimal::zero()))
.unwrap_or_else(|_| BigDecimal::zero());
```